Area chart changes (#209)
patternfly/training-scenarios: react-charts/area-chart/step7.md

> In this step you will change the area chart theme color.

Reviewer: Let's try to remove the instances of "you will" when we can. So for example, this could read, "Change the area chart theme color."

1) <strong>Make sure the `App.js` file is still open.</strong>

2) <strong>Locate the `<Chart>` component and add a property to change the theme.</strong>

> The `themeColor` property specifies the theme color. Valid values are 'blue,' 'green,' 'multi,' etc. It should look like the following:

Reviewer 2: Since we aren't showing all of the valid values, should we reword and also use the enum values instead of strings:
```suggestion
The `themeColor` property specifies the theme color. Some valid values are `ChartThemeColor.blue`,  `ChartThemeColor.green`, or `ChartThemeColor.multi`. It should look like the following:
```
